src/main/java/hudson/plugins/buildblocker/BuildBlockerQueueTaskDispatcher.java

```diff
@@ -139,24 +139,25 @@ private CauseOfBlockage checkForBlock(Node node, Queue.Item item, BuildBlockerPr
         return null;
     }
 
-    private SubTask checkAccordingToProperties(Node node, Queue.Item item, BuildBlockerProperty properties) {
+    @WithBridgeMethods(value=SubTask.class)
```

Comment:
:ant: You do not need to use compatibility annotations on `private` (or package-scope) methods, as no other plugin could have been using them to begin with.